.env.example

```
@@ -27,11 +27,26 @@ DAO_FACTORY_ADDRESS="0xB815791c233807D39b7430127975244B36C19C8e"
 PLUGIN_REPO_FACTORY_ADDRESS="0x399Ce2a71ef78bE6890EB628384dD09D4382a7f0"
 
 # Token Settings
+# Leave TOKEN_ADDRESS unset (or 0x0) to deploy a fresh GovernanceERC20 and mint it per the
+# MINT_SETTINGS_* below. Set it to an existing token to reuse that one instead: if the token
+# implements IVotes it is used directly, otherwise it is wrapped in a GovernanceWrappedERC20.
+# Reusing one token is how several plugins (e.g. a public and a private process) end up sharing
+# the same voting power and delegation.
+TOKEN_ADDRESS=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)